python/doc/math_notations.sty

```diff
@@ -48,11 +48,9 @@
 
 % A set
 \newcommand{\set}[1]{\mathcal{#1}}
-% The set of standard input variables
-\newcommand{\standardInputSpace}{\set{Z}}
```

Comment:
This is surprising, because removing this macro does not brakes any help page. It might break [functional_chaos.html](http://openturns.github.io/openturns/latest/theory/meta_modeling/functional_chaos.html), but the macro is not used here.

Reply:
I checked whether the command was used... it was not!